gp-includes/template.php

```diff
@@ -243,6 +243,30 @@ function gp_select( $name_and_id, $options, $selected_key, $attrs = array() ) {
 	$attributes = $attributes ? " $attributes" : '';
 	$res = "<select name='" . esc_attr( $name_and_id ) . "' id='" . esc_attr( $name_and_id ) . "' $attributes>\n";
 	foreach( $options as $value => $label ) {
+		$label_hidden = [
+			'hidden' => _x( 'hidden', 'Priority', 'glotpress' ),
+		];
+		$label_low = [
+			'low' => _x( 'low', 'Priority', 'glotpress' ),
+		];
+		$label_normal = [
+			'normal' => _x( 'normal', 'Priority', 'glotpress' ),
+		];
+		$label_high = [
+			'high' => _x( 'high', 'Priority', 'glotpress' ),
+		];
```

Comment:
All the `$label_xx` arrays should be combined into one `$labels` array otherwise it doesn't make sense to have an array with just one value.

The array should also be defined before the `foreach` loop.

Reply:
Done in https://github.com/GlotPress/GlotPress-WP/pull/813/commits/2fbaef31ed66b3cdf24de835954ffa71444e43a0